src/applications/caregivers/containers/App.jsx

```diff
@@ -15,23 +15,26 @@ const App = ({ loading, location, children }) => {
   useEffect(
     () => {
       if (!loading) {
-        const checkBoxes = [
-          ...document.querySelectorAll('[value="Y"]'),
-          ...document.querySelectorAll('[value="N"]'),
-        ];
-        for (const checkbox of checkBoxes) {
-          checkbox.onclick = e => {
-            const label = e.target.closest('fieldset')?.firstChild?.innerText;
+        const radios = document.querySelectorAll('input[type="radio"]');
```

Comment:
I don't understand the problem here well enough but it seems a little risky to me to bind onclick handlers to all radio boxes on the page